contracts/token/erc721/erc721psi/ERC721Psi.sol

```diff
@@ -0,0 +1,571 @@
+// SPDX-License-Identifier: MIT
+/**
+  ______ _____   _____ ______ ___  __ _  _  _ 
+ |  ____|  __ \ / ____|____  |__ \/_ | || || |
+ | |__  | |__) | |        / /   ) || | \| |/ |
+ |  __| |  _  /| |       / /   / / | |\_   _/ 
+ | |____| | \ \| |____  / /   / /_ | |  | |   
+ |______|_|  \_\\_____|/_/   |____||_|  |_|   
+
+ - github: https://github.com/estarriolvetch/ERC721Psi
+ - npm: https://www.npmjs.com/package/erc721psi
+                                          
+ */
+
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/token/ERC721/IERC721.sol";
+import "@openzeppelin/contracts/token/ERC721/IERC721Receiver.sol";
+import "@openzeppelin/contracts/token/ERC721/extensions/IERC721Metadata.sol";
+import "@openzeppelin/contracts/utils/Context.sol";
+import "@openzeppelin/contracts/utils/Strings.sol";
+import "@openzeppelin/contracts/utils/introspection/ERC165.sol";
+import "@openzeppelin/contracts/utils/Address.sol";
+import "@openzeppelin/contracts/utils/StorageSlot.sol";
+import "solidity-bits/contracts/BitMaps.sol";
+
+contract ERC721Psi is Context, ERC165, IERC721, IERC721Metadata {
```

Comment:
removed `tokensOfOwner` from original implementation since thats not supported by ERC721